resources/css/main.css

```diff
@@ -1,37 +1,499 @@
+}
+
+.editors-pick {
+  clear: both;
+  margin-bottom: 15px;
+  cursor: pointer;
+}
+
+.editors-pick-img {
+  float: left;
+  margin-right: 10px;
+  margin-bottom: 15px;
+  width: 75px;
+  height: 75px;
+}
+
+.editors-pick-title {
+  display: block;
+  color: #333;
+  font-size: 13px;
+  font-family: Georgia, 'Times New Roman', Times, serif;
+  font-weight: 400;
+  padding-top: 20px;
+}
+
+.editors-pick-title:hover,
+.editors-pick-title:focus {
+  text-decoration: underline;
+}
+
+/** Sharing Toolbar **/
+.social-toolbar {
+  width: 600px;
+  margin: 0 auto;
+  border-top: 1px solid rgb(235, 235, 235);
+  padding-top: 20px;
+  padding-bottom: 20px;
+}
+
+.social-toolbar-list {
+  list-style: none;
+}
+
+.social-toolbar-list-item {
+  display: inline-block;
+  margin-right: 14px;
+}
+
+.social-share {
+  background-color: rgb(238, 238, 238);
+  border-radius: 100%;
+  width: 15px;
+  height: 15px;
+  padding: 5px;
+  box-sizing: content-box;
+  border: 2px solid rgb(238, 238, 238);
+}
+
+.social-share:hover,
+.social-share:focus {
+  background-color: white;
+}
+
+.non-social {
+  background-color: white;
+  position: relative;
+  bottom: 2px;
+}
+
+.non-social:hover,
+.non-social:focus {
+  opacity: 0.2;
+}
+
+.comments-btn {
+  background: none;
+  border: 1px solid #666;
+  font-family: Arial, Helvetica, sans-serif;
+  border-radius: 3px;
+  padding: 5px 8px;
+  position: relative;
+  top: -10px;
+  cursor: pointer;
+}
+
+.comments-btn:hover,
+.comments-btn:focus {
+  border-color: #eee;
+}
+
+/** Inflation **/
+.inflation {
+  width: 945px;
+  margin: 0 auto;
+  margin-top: 15px;
+}
+
+.inflation-header-title {
+  font-family: georgia, "times new roman", serif;
+  font-style: normal;
+  font-size: 17px;
+  font-weight: 700;
+  margin-bottom: 2px;
+}
+
+.inflation-header-subtitle {
+  color: rgb(102, 102, 102);
+  font-family: Georgia, 'Times New Roman', Times, serif;
+  font-style: normal;
+  font-size: 14px;
+  font-weight: 400;
+}
+
+.inflation-container {
+  display: flex;
+  flex-wrap: wrap;
+  margin-top: 15px;
+}
+
+.inflation-container-section {
+  width: 25%;
+  padding-right: 15px;
+  padding-bottom: 15px;
+}
+
+.inflation-container-section-text {
+  font-family: arial, helvetica, sans-serif;
+  font-size: 13px;
+  font-weight: 300;
+  font-style: normal;
+  line-height: 17px;
+  color: black;
+}
+
+.inflation-container-section-first-phrase {
+  font-family: inherit;
+  font-size: 11px;
+  font-weight: bold;
+  letter-spacing: 0.05em;
+}
+
+.inflation-footer::before {
+  margin-bottom: 13px;
+  background-color: black;
+  display: inline-block;
+  width: 60px;
+  height: 1px;
+  content: "";
+}
+
+.inflation-footer-authors {
+  color: rgb(51, 51, 51);
+  font-family: arial, helvetica, sans-serif;
+  font-size: 13px;
+}
+
+/** Site index **/
+#site-index {
+  font-family: Arial, Helvetica, sans-serif;
+  width: 1200px;
+  margin: 90px auto 10px;
+  border-top: 1px solid #e2e2e2;
+}
+
+#site-index::before {
+  content: "";
```

Comment:
- [ ] You have empty properties

Reply:
This is intentional. The `content = "";` is meant to create a shallow top border for the site index. It is that way in the New York Times article as well.